src/constants/__tests__/__snapshots__/snippets.ts.snap

```diff
@@ -6,6 +6,7 @@ Array [
   "Cxs",
   "Glamorous",
   "Radium",
+  "ReactJss",
```

Comment:
I normally write JSS, because its an abbreviation, probably same goes for CXS, but if you like it this way, its ok.

Reply:
I think what I'll probably end up doing is dropping JSS, and then just use React-JSS as JSS if that makes sense? Thoughts?

Reply:
I would love to see both, React-JSS and JSS, because they are abstractions on different layers. It demostrates the flexibility, because JSS has no dependency to rendering library and react-jss implements stuff like theming and sheets reuse.

Reply:
I can do that!